lib/reporter/index.js

```diff
@@ -180,13 +180,17 @@ class Reporter extends SimplifiedReporter {
     try {
       let commandResult = result;
 
-      const isSuccess =  result === undefined ||
+      let isSuccess =  result === undefined ||
         result == null ||
         !((result instanceof Error) ||
           (result.error instanceof Error) ||
           result.status === -1
         );
 
+      if (node.fullName === 'waitForElementNotPresent') {
```

Comment:
can we find a more elegant way to do this?

Reply:
thinking..

Reply:
@beatfactor I am passing a new flag in results now. Does this look okay?